flowexperimental/comp/flowexp_comp.cpp

```diff
@@ -19,18 +19,58 @@
 #include "config.h"
 
 #include <opm/input/eclipse/Deck/Deck.hpp>
-#include <opm/input/eclipse/Parser/Parser.hpp>
 #include <opm/input/eclipse/Parser/ParseContext.hpp>
+#include <opm/input/eclipse/Parser/Parser.hpp>
 
 #include <opm/models/utils/start.hh>
 
 #include <opm/simulators/flow/FlowGenericProblem_impl.hpp>
 
 #include <fmt/format.h>
 
+#include <tuple>
+
 #include "flowexp_comp.hpp"
 
-int main(int argc, char** argv)
+template <int compileTimeComponent>
+std::tuple<bool, int>
+runComponent(int runtimeComponent, int argc, char** argv)
+{
+    if (runtimeComponent == compileTimeComponent) {
+        return std::make_tuple(true, Opm::dispatchFlowExpComp<compileTimeComponent>(argc, argv));
+    }
+    return std::make_tuple(false, EXIT_FAILURE);
```

Comment:
Tiny nit: You need to `#include <cstdlib>` to bring a definition of the `EXIT_FAILURE` macro into scope.